doc/source/user_guide/indexing.rst

```diff
@@ -105,7 +105,7 @@ Basics
 ------
 
 As mentioned when introducing the data structures in the :ref:`last section
-<basics>`, the primary function of indexing with ``[]`` (a.k.a. ``__getitem__``
+<basics>`, the primary function of indexing with ``[]`` (a.k.a ``__getitem__``
```

Comment:
You don't need to change this in the user guide. The issue only comes up in the first line of docstrings